metadata-models/src/main/pegasus/com/linkedin/metadata/key/SchemaFieldKey.pdl

```diff
@@ -0,0 +1,27 @@
+namespace com.linkedin.metadata.key
+
+import com.linkedin.common.Urn
+
+/**
+ * Key for a SchemaField
```

Comment:
Should we call these models DatasetSchemaField?

Reply:
hmm.. actually updating this to be `container` rather than `dataset` so we have the flexibility to add schema fields to other things